src/layouts/header.scss

```diff
@@ -95,7 +95,7 @@ $lg: "screen and (max-width : 1024px)";
     }
 
     &:hover {
-      color: #5f4b0f;
+      color: rgb(75, 75, 75);
```

Comment:
This is a revert from https://github.com/cilium/ebpf.io/pull/127.
Can you please tell why you are reverting the change?